### DISCUSSION TITLE
blanko preset

### DISCUSSION THREAD

**Discussion by woldtwerk** [wingsuit-designsystem/wingsuit#209] [Q&A] [answered]
Is it currently possible to have a blanko preset?

Meaning without:
```
defaultPresets: {
    babel,
    assets,
    assetsVideos,
    svg,
    twing,
    storybook,
    drupal,
    cms,
  },
  presets: [css, assets, assetsVideos, svg],
```

Goal is to only compile twig and yml files with wingsuit and handle css, js, assets externally.

**Comment by christianwiedemann** [comment, marked as answer]
Hi, 
I think the easiest solution is to create an own preset and publish this preset to npm. Should be easy. Checkout the pages preset
https://github.com/wingsuit-designsystem/wingsuit/blob/master/presets/pages/src/index.ts. 

You propably need one for storybook and one for drupal. Here is the base configuration. 
https://github.com/wingsuit-designsystem/wingsuit/blob/master/packages/core/src/stubs/defaultWingsuitConfig.stub.ts
